packages/utils/test/utils.spec.ts

```diff
@@ -0,0 +1,131 @@
+import { SchemaConfig } from '@s1seven/schema-tools-types';
+import { readFileSync } from 'fs';
+import {
+  asECoCCertificate,
+  asEN10168Certificate,
+  getRefSchemaUrl,
+  getSchemaConfig,
+  getTranslations,
+} from '../src/index';
+import { cache, loadExternalFile } from '../src/index';
+import { Stream } from 'stream';
+import { URL } from 'url';
+import axios from 'axios';
+jest.mock('axios');
+
+const mockCacheGet = jest.fn();
+const mockCacheSet = jest.fn();
+(cache as any) = { get: mockCacheGet, set: mockCacheSet };
+
+const EN_CERT_PATH = `${__dirname}/../../../fixtures/EN10168/v0.0.2/valid_cert.json`;
+const ECOC_CERT_PATH = `${__dirname}/../../../fixtures/E-CoC/v0.0.2-2/valid_cert.json`;
+const MOCK_CERT = 'cert';
+const enSchema = JSON.parse(readFileSync(EN_CERT_PATH, 'utf8') as string);
+const refSchemaUrl = new URL('https://schemas.en10204.io/en10168-schemas/v0.0.2/schema.json');
+const schemaConf: SchemaConfig = {
+  baseUrl: 'https://schemas.en10204.io',
+  schemaType: 'en10168-schemas',
+  version: '0.0.2',
+};
+const certificateLanguages = ['DE', 'FR', 'EN'];
+
+describe('Utils', function () {
+  describe('E-CoC', function () {
+    const ecocSchema = JSON.parse(readFileSync(ECOC_CERT_PATH, 'utf8') as string);
+
+    it('should validate a valid certificate', async () => {
+      const responseOk = asECoCCertificate(ecocSchema, '/');
+      expect(responseOk).toHaveProperty('ok');
+    });
+
+    it('should validate an invalid certificate', async () => {
+      delete ecocSchema.RefSchemaUrl;
+      const responseInvalid = asECoCCertificate(ecocSchema, '/');
+      expect(responseInvalid).toHaveProperty('error');
+    });
+  });
+
+  describe('EN10168', function () {
+    it('should validate a valid certificate', async () => {
+      const responseOk = asEN10168Certificate(enSchema, '/');
+      expect(responseOk).toHaveProperty('ok');
+    });
+
+    it('should validate an invalid certificate', async () => {
+      delete enSchema.Certificate;
+      const responseInvalid = asEN10168Certificate(enSchema, '/');
+      expect(responseInvalid).toHaveProperty('error');
+    });
+  });
+
+  describe('loadExternalFile()', function () {
+    const mockCacheGet = jest.fn();
+    const mockCacheSet = jest.fn();
+    (cache as any) = { get: mockCacheGet, set: mockCacheSet };
+    beforeEach(() => {
+      mockCacheGet.mockClear();
+      mockCacheSet.mockClear();
+    });
+
+    it('should load file from cache', async () => {
+      mockCacheGet.mockReturnValueOnce(enSchema);
+      const certificate = await loadExternalFile(EN_CERT_PATH, 'json');
```

Comment:
I know i sent you the snippet with that variable name `certificate`, but it's a schema that should be returned here.
So could you rename those variables to avoid confusions please ?